docs/content/guide/$location.ngdoc

```diff
@@ -227,18 +227,18 @@ it('should show example', inject(
   },
   function($location) {
     // open http://example.com/base/index.html#!/a
-    $location.absUrl() == 'http://example.com/base/index.html#!/a'
```

Comment:
This whole example/test is wrong anyway. But it's not related to this PR, so feel free to leave as is.